src/AuthenticationServiceInterface.php

```diff
@@ -56,25 +56,6 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
      */
     public function getIdentity();
 
-    /**
-     * Clears the identity from authenticators that store them and the request.
-     *
-     * @param \Psr\Http\Message\ServerRequestInterface $request The request.
-     * @param \Psr\Http\Message\ResponseInterface $response The response.
-     * @return array Return an array containing the request and response objects.
-     */
-    public function clearIdentity(ServerRequestInterface $request, ResponseInterface $response);
```

Comment:
`clearIdentity` and `setIdentity` were removed in favor of `PersistenceInterface` which now `AuthenticationServiceInterface` extends.